File: packages/react-native-reanimated/android/src/main/java/com/swmansion/reanimated/keyboard/KeyboardWorkletWrapper.java
package com.swmansion.reanimated.keyboard;

import com.facebook.jni.HybridData;
import com.facebook.proguard.annotations.DoNotStrip;

@DoNotStrip
public class KeyboardWorkletWrapper {
  @DoNotStrip private final HybridData mHybridData;

  @DoNotStrip
  private KeyboardWorkletWrapper(HybridData hybridData) {
    mHybridData = hybridData;
  }

  public native void invoke(int keyboardState, int height);
}

File: packages/react-native-reanimated/android/src/main/java/com/swmansion/reanimated/keyboard/KeyboardWorkletWrapper.kt
package com.swmansion.reanimated.keyboard

import com.facebook.jni.HybridData
import com.facebook.proguard.annotations.DoNotStrip

@DoNotStrip
class KeyboardWorkletWrapper {
  @field:DoNotStrip private val mHybridData: HybridData

  @DoNotStrip
  private constructor(hybridData: HybridData) {
    mHybridData = hybridData
  }

  external fun invoke(keyboardState: Int, height: Int)
}
